frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -137,16 +138,16 @@ class PlaneController extends React.PureComponent<Props> {
     // newest input catchers. We should refactor the
     // InputMouse handling so that this is not necessary anymore.
     // See: https://github.com/scalableminds/webknossos/issues/3475
-    const tdSelector = `#inputcatcher_${OrthoViews.TDView}`;
-    Utils.waitForSelector(tdSelector).then(() => {
+    const tdSelector = `inputcatcher_${OrthoViews.TDView}`;
```

Comment:
Looking at the above comment, could it be that this is no longer an issue now? If so, we could remove the code waiting for the td view (line 142) and the comment :)